src/attestation/Attestation.ts

```diff
@@ -162,15 +185,15 @@ export default class Attestation implements IAttestation {
    * });
    * ```
    */
-  public async verify(claimHash: string = this.claimHash): Promise<boolean> {
+  public static async verify(
```

Comment:
good suggestion
As to the wrapping: in itself it would be usefull for not typing the whole class as with static, and just accessing the wrapper function.
But here verify needs Chain connection, have to talk to @tjwelde maybe about that in general.